README.md

```diff
@@ -1,47 +1,43 @@
-extract the Pega distribution to the same directory as the Dockerfile.  It is recommended that this is done on a Linux system to retain proper file permissions.  Replace the source paths with the actual paths to the Pega 8 software libraries and specify a valid JDBC driver for your target database.
+The simplest way to use this image is to create your own Dockerfile with contents similar to the example below and extract the Pega distribution to the same directory as the Dockerfile.  It is recommended that this is done on a Linux system to retain proper file permissions.  Replace the source paths with the actual paths to the Pega Infinity software libraries and specify a valid JDBC driver for your target database.
 
-    FROM pegasystems/docker-pega-web-ready
+    FROM pegasystems/pega-ready
     
     # Expand prweb to target directory
-    COPY archives/prweb.war /opt/pega/prweb.war
-    RUN unzip -q -d /opt/pega/prweb /opt/pega/prweb.war
+    COPY archives/prweb.war ${CATALINA_HOME}/webapps/prweb.war
+    RUN unzip -q -o -d ${CATALINA_HOME}/webapps/prweb ${CATALINA_HOME}/webapps/prweb.war && \
+    rm -rf ${CATALINA_HOME}/webapps/prweb.war
 
     # Make jdbc driver available to tomcat applications
-    COPY /path/to/jdbcdriver.jar /usr/local/tomcat/lib/
+    COPY /path/to/jdbcdriver.jar ${CATALINA_HOME}/lib/
 
 Build the image using the following command:
 
-    docker build -t pega8-tomcat .
+    docker build -t pega-tomcat .
 
 Upon successful completion of the above command, you will have a Docker
-image that is registered in your local registry named pega8-tomcat:latest
-and that you can view using the `docker images` command
+image that is registered in your local registry named pega-tomcat:latest
+and that you can view using the `docker images` command.
+
+## Running the image
+
+When running Pega using containers, an orchestration tool is required.  Pega provides support for deployments on Kubernetes using either Helm charts or direct yaml files.  Source code for the deployment scripts may be found in the [pega-helm-charts](https://github.com/pegasystems/pega-helm-charts) repository and more information about Pega on client managed cloud may be found on the [Cloud Choice](https://community.pega.com/knowledgebase/articles/cloud-choice) community site.
 
-# Customizations
+## Image customizations
 
-**JDK and Tomcat**
+**Base image**
 
-Currently the DockerFile extends the base image `tomcat:9-jre11` . This has been tested internally and works fine.
-You can change this to use your preferred Tomcat base image. However this should be thoroughly tested and verfied at your end.There may be some compatibility issue on changing the base image.
+This Docker image extends the base image `tomcat:9-jre11`. This has been extensively validated. You may choose change this to use your preferred Tomcat base image, however any change should be thoroughly tested and verfied.
```

Comment:
extensively validated ==> thoroughly validated?? looks odd next to "extends the base image ;-)